src/components/Projects/ProjectTabs.js

```diff
@@ -20,64 +21,65 @@ const ProjectTabs = ( ): Node => {
   const memberId = currentUser?.id;
   const [apiParams, setApiParams] = useState( { } );
   const [currentTabId, setCurrentTabId] = useState( JOINED_TAB_ID );
-  const latLng = useUserLocation( );
+  const { latLng, isLoading: isLoadingLocation } = useUserLocation( { skipPlaceGuess: true } );
 
   const {
-    data: projects
+    data: projects,
+    isLoading
   } = useAuthenticatedQuery(
     ["searchProjects", apiParams],
     optsWithAuth => searchProjects( apiParams, optsWithAuth )
   );
 
-  const fetchProjectsByLatLng = ( ) => {
-    setApiParams( {
-      lat: latLng.latitude,
-      lng: latLng.longitude
-    } );
-  };
-
-  const fetchFeaturedProjects = ( ) => setApiParams( { features: true } );
-  const fetchUserJoinedProjects = useCallback( ( ) => setApiParams(
-    { member_id: memberId }
-  ), [memberId] );
-
   useEffect( ( ) => {
-    if ( memberId ) {
-      fetchUserJoinedProjects( );
+    if ( memberId && currentTabId === JOINED_TAB_ID ) {
+      setApiParams( { member_id: memberId } );
+    } else if ( currentTabId === FEATURED_TAB_ID ) {
+      setApiParams( { features: true } );
+    } else if ( currentTabId === NEARBY_TAB_ID ) {
+      if ( latLng ) {
```

Comment:
I think you could just add this to the `else if` above, e.g. `} else if ( currentTabId === NEARBY_TAB_ID && latLng ) {`